packages/database/src/core/snap/ChildrenNode.ts

```diff
@@ -324,7 +324,10 @@ export class ChildrenNode implements Node {
   /**
    * @inheritDoc
    */
-  forEachChild(index: Index, action: (key: string, node: Node) => void): any {
+  forEachChild(
+    index: Index,
+    action: (key: string, node: Node) => boolean | void
+  ): any {
```

Comment:
should the return type be boolean instead of any?

Reply:
Yeah, good call. I changed this as well.